jina/orchestrate/flow/base.py

```diff
@@ -1479,7 +1487,8 @@ def start(self):
 
         self._build_level = FlowBuildLevel.RUNNING
 
-        send_telemetry_event(event='start', obj=self)
+        self._start_time = time.time()
+        send_telemetry_event(event='start', obj=self, entity_id=self._entity_id)
```

Comment:
I don't understand why do we need to define _entity_id and send start event in Flow ? isn't this already handled in asyncio runtime when it's a gateway ?